WordPress/Classes/ViewRelated/Media/SiteMedia/Views/SiteMediaCollectionCellSelectionOverlayView.swift

```diff
@@ -1,4 +1,5 @@
 import UIKit
+import Aztec
```

Comment:
If possible, let's remove the use of the `Aztec ` public APIs in places where it shouldn't be used.

Reply:
I can create a follow up PR for that, but it's probably out of scope for this one.

Reply:
Here is the PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/23500